.github/workflows/ping-interop-all.yml

```diff
@@ -0,0 +1,45 @@
+on:
```

Comment:
Instead of introducing a new workflow file, how about including nim in `ping-interop.yml`. I don't see a reason why we should treat Go or Rust special.

Maybe @MarcoPolo has opinions here.

Reply:
`ping-interop.yml` will call the `go-rust-interop.toml` composition
So it could just call the `all-interop.toml` instead, but then the `go-rust-interop.toml` will never be called, and maybe it should be removed?
We would just have:
- `[language].toml`: list of versions of that implem
- `[language]-cross-versions.toml`: test every version of that implem
- `all-interop-latest.toml`: test last version of every implem
- `all-interop.toml`: test every version of every implem

Reply:
Did this in my latest commit, we can revert if not happy

Reply:
> We would just have:
> 
>     * `[language].toml`: list of versions of that implem
> 
>     * `[language]-cross-versions.toml`: test every version of that implem
> 
>     * `all-interop-latest.toml`: test last version of every implem
> 
>     * `all-interop.toml`: test every version of every implem

On a high level I am in favor of this. I will take a detailed look now.